src/components/HandoffLink/HandoffLink.test.tsx

```diff
@@ -28,6 +28,7 @@ describe('HandoffLink', () => {
         }}
       >
         <HandoffLink path="/contacts">
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
Could you just define this once at the top of the file to disable it for the whole file?

Reply:
In general, I like having the rules enabled for the rest of the file and like having the line specific ones go away when the line is deleted. If they are up top, they usually last forever.

But this specific file is a test. Maybe we don't need to enforce a11y rules on test files. Also I expect this file specifically to be deleted at the end of the rewrite.